keylime/ca_impl_cfssl.py

```diff
@@ -68,6 +69,8 @@ def post_cfssl(params,data):
     return response.json()
 
 def start_cfssl(cmdline=""):
+    if shutil.which("cfssl") is None:
+        raise Exception("cfssl binary not found in the path.  Please install cfssl or change the setting \"ca_implementation\" in keylime.conf")
```

Comment:
I guess this is a personal thing, but could we use `logger.error` followed by an `exit(1)`:

```
logger.error("cfssl binary not found in the path.  Please install cfssl or change the setting \"ca_implementation\" in keylime.conf")
exit(1)
```

It then looks like a gracefully captured error.